README.md

```diff
@@ -12,6 +12,10 @@
 
 Before we start, make sure your computer has R and Python development environment set up. IDEs like [R Studio](https://posit.co/products/open-source/rstudio/) and [Visual Studio Code](https://code.visualstudio.com/) are optional but recommended.
 
+The following flowchart gives an overview for the script workflow:
+
+![Figure 1, Flowchart for scripts and workflow](flowchart.png)
```

Comment:
I think you may have generated with diagrams.net (i.e., draw.io)? If so, it might be a good idea to include the source file here as well.

Reply:
Perhaps as `flowchart.drawio`?